Geometry/HGCalCommonData/src/HGCalDDDConstants.cc

```diff
@@ -1241,15 +1246,15 @@ std::pair<double, double> HGCalDDDConstants::waferPosition(int waferU, int wafer
 int HGCalDDDConstants::waferType(DetId const& id) const {
```

Comment:
The return value of this function is the previous one minus 1. This is correctly propagated to the HGCalDigitizer, the only users of this method as far as I could tell: therefore the net effect is that no changes are expected. (OK!)

Reply:
This function has not changed at all except it uses enum values instead of numbers. The o/p is indeed unchanged